### PR TITLE
AWS Glue Catalog doc

### PR DESCRIPTION
**Pull request by aDaemonThread**

### CODE REVIEW

File: src/reference/config/catalogs/catalog-aws-glue.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+category:
+  - Catalog
+---
+
+# aws-glue Catalog
+
+[Available in <ZillaPlus/>](https://www.aklivity.io/products/zilla-plus)
+{.zilla-plus-badge .hint-container .info}
+
+Zilla runtime aws-glue catalog
```
Comment (vordimous) [inline src/reference/config/catalogs/catalog-aws-glue.md:14]:
```suggestion
Zilla runtime aws-glue catalog.
```

File: src/reference/config/catalogs/catalog-aws-glue.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+> `number` | Default: `"300"`
+
+Configure the Time-To-Live for the schema information retrieved against latest version. Defaults to 5 minutes. 
+
+#### options.compression
+
+> `enum` [ "none", "zlib" ] | Default: `"none"`
+
+Configuration to set compression level.
```
Comment (vordimous) [inline src/reference/config/catalogs/catalog-aws-glue.md:64]:
```suggestion
Configures the compression level for fetched schemas.

```

File: src/reference/config/catalogs/catalog-aws-glue.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+### options
+
+> `object`
+
+#### options.registry\*
+
+> `string`
+
+AWS Glue Registry name to access schemas.
```
Comment (vordimous) [inline src/reference/config/catalogs/catalog-aws-glue.md:52]:
```suggestion
The AWS Glue Registry name to access schemas.
```

File: src/reference/config/catalogs/catalog-aws-glue.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+#### options.max-age
+
+> `number` | Default: `"300"`
+
+Configure the Time-To-Live for the schema information retrieved against latest version. Defaults to 5 minutes.
```
Comment (vordimous) [inline src/reference/config/catalogs/catalog-aws-glue.md:58]:
```suggestion
Configures the time to live in `seconds` for the schema information retrieved against the latest version. The default is 300 seconds or 5 minutes. 
```

File: src/reference/config/catalogs/catalog-aws-glue.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  type: aws-glue
+  options:
+    registry: zilla
+    max-age: 30
+    compression: zlib
+```
+
+## Summary
+
+Defines a catalog with a schema pulled from aws glue to enforce validation.
```
Comment (vordimous) [inline src/reference/config/catalogs/catalog-aws-glue.md:27]:
```suggestion
Defines a catalog that can fetch schemas [AWS Glue](https://aws.amazon.com/glue/) to enforce validation.
```

**Review by vordimous — CHANGES_REQUESTED** [review]

File: src/reference/config/catalogs/catalog-aws-glue.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+AWS Glue Registry name to access schemas.
+
+#### options.max-age
+
+> `number` | Default: `"300"`
+
+Configure the Time-To-Live for the schema information retrieved against latest version. Defaults to 5 minutes. 
+
+#### options.compression
```
Comment (vordimous) [inline src/reference/config/catalogs/catalog-aws-glue.md:60]:
```suggestion
#### options.model-compression
```

I think the `model-` prefix adds just enough information to tie it to the model-view implementations. It also reads as a global setting for all models, which we can clarify in the property description. 

@jfallows @aDaemonThread

Comment (jfallows) [reply src/reference/config/catalogs/catalog-aws-glue.md:60]: @vordimous Let's go with what we have for now, as this is already released in Zilla Plus and we need to get the docs out to let folks be successful with it.

We can consider an evolution in this syntax for a future Zilla Plus release.

**Review by vordimous — APPROVED** [review]

**Pull request merged**